frontend/src/metabase/admin/datamodel/containers/MetricForm.jsx

```diff
@@ -57,7 +57,7 @@ export default class MetricForm extends Component {
     this.props.updatePreviewSummary({
       ...datasetQuery,
       query: {
-        aggregation: ["count"],
+        aggregation: [["count"]],
         ...datasetQuery.query,
       },
     });
```

Comment:
This will have conflicts with #10732, but I can resolve them whenever we're merging.